src/OnlineRetailPortal.Contracts/Contracts/UploadImageRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts.Contracts
+{
+    public class UploadImageRequest
+    {
+        public IFormFile File { get; set; }
+    }
+    public class UploadImageRequestValidator : AbstractValidator<UploadImageRequest>
```

Comment:
move this to different class, also what happens when the property is null ?? are we throwing exception ?